.github/workflows/test-java.yml

```diff
@@ -0,0 +1,62 @@
+name: test-java
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build-javascript:
+    uses: cucumber/html-formatter/.github/workflows/build-javascript.yml@github-ci
+
+  test-java:
+    needs: [build-javascript]
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        java: ['11', '16']
+        include:
+          - os: windows-latest
+            java: '16'
+          - os: macos-latest
+            java: '16'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Download javascript build artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: javascript-dist
+          path: javascript/dist
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java }}
+          cache: 'maven'
+
+      - name: Copy javascript artifacts into target classes
+        working-directory: java
+        run: |
+          mkdir -p target/classes/io/cucumber/htmlformatter
+          cp ../javascript/dist/main.css target/classes/io/cucumber/htmlformatter/cucumber-html.css
+          cp ../javascript/dist/main.js target/classes/io/cucumber/htmlformatter/cucumber-html.js
+          cp ../javascript/dist/src/index.mustache.html target/classes/io/cucumber/htmlformatter/index.mustache.html
```

Comment:
Great to get this working!

I wonder if it will be awkward to work on the Java implementation without some locally-run automation for these preparatory steps. Maybe it doesn't matter and we can deal with that problem if / when it comes up.

Reply:
https://github.com/cucumber/html-formatter/issues/3